cross/opencore-amr/Makefile

```diff
@@ -0,0 +1,29 @@
+PKG_NAME = opencore-amr
+PKG_EXT = tar.gz
+PKG_VERS = 0.1.3
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://sourceforge.net/projects/opencore-amr/files/opencore-amr
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)/
+
+
+DEPENDS =
+
+HOMEPAGE = https://github.com/Distrotech/opencore-amr/
+COMMENT  = OpenCORE Adaptive Multi Rate (AMR) speech codec library implementation.
+LICENSE  = Apache 2
+
+GNU_CONFIGURE = 1
+
+PRE_CONFIGURE_TARGET = myPreConfigure
+
+ifeq ($(findstring $(ARCH), $(ARM5_ARCHES)),$(ARCH))
```

Comment:
Throughout the whole PR, sections like these are added below the `include`. 
If it works like this (above the include), let's use this format instead. I recall a discussion on this before, no idea where though :/